acceptance/install/scenario4_test.go

```diff
@@ -143,6 +143,18 @@ var _ = Describe("<Scenario4> EKS, epinio-ca, on S3 storage", func() {
 			Expect(err).ToNot(HaveOccurred(), out)
 		})
 
+		By("Workarounds for EKS 1.24+", func() {
+			// Set a ingress annotation to make ELB and nginx-ingress compatible with websocket
+			out, err := proc.Kubectl("annotate", "ingress", "--namespace", "epinio", "epinio", "nginx.org/websocket-service='epinio-server'")
```

Comment:
Is this annotation something we should add to our helm chart ?
Being AWS specific, would its general presence break other providers ?

Reply:
Beyond this question I like it.

Reply:
I believe this can be added into our helm chart. I don't expect it will break other ingress providers but to be sure I'm testing now with k3s and traefik...

Reply:
@andreas-kupries WebUI as well as Logs streaming for Staging job in CLI works ok with this annotation on k3s with traefik.

Reply:
We are already mixing traefik and nginx annotations anyway on epinio ingress resource:
```
  annotations:
    meta.helm.sh/release-name: epinio
    meta.helm.sh/release-namespace: epinio
    nginx.ingress.kubernetes.io/proxy-body-size: 100m
    nginx.ingress.kubernetes.io/ssl-redirect: "true"
    traefik.ingress.kubernetes.io/router.entrypoints: websecure
    traefik.ingress.kubernetes.io/router.tls: "true"
```

Reply:
Ok. If you have not started yet I can make a chart PR adding this annotation.

Reply:
... And done :arrow_right_hook:  https://github.com/epinio/helm-charts/pull/376

Reply:
Thanks, I will remove the annotation part then.

Reply:
Merged. This PR can include it now, and possibly remove the kubectl call above.